libmcount/dynamic.c

```diff
@@ -258,6 +258,7 @@ __weak void mcount_arch_patch_branch(struct mcount_disasm_info *info, struct mco
 
 struct find_module_data {
 	struct uftrace_sym_info *sinfo;
+	bool skip_first; /* fist module is the main binary */
```

Comment:
I'm not sure if it's guaranteed that the first module is the main binary especially on AArch64/Android.

Reply:
This is now fixed